### ISSUE TITLE
Python client exception using api key for grpc

### ISSUE THREAD

**Issue by IvanPleshkov** [qdrant/qdrant-client#228]
## Current Behavior
Python client panics when api key is used with grpc. For rest api it works fine.

## Steps to Reproduce
1. Run QDrant
```
docker run -p 6333:6333 -p 6334:6334 \
    -e QDRANT__SERVICE__GRPC_PORT="6334" \
    -e QDRANT__SERVICE__API_KEY="key" \
    -v ./storage:/qdrant/storage \
    -v ./snapshots:/qdrant/snapshots \
    qdrant/qdrant:latest
```
2. Run Python script:
```
from qdrant_client import QdrantClient

qdrant_client = QdrantClient(url="http://localhost:6333",prefer_grpc=True,timeout=30,api_key="key") # this one didnt work
# with rest everything is fine:
# qdrant_client = QdrantClient(url="http://localhost:6333",timeout=30,api_key="key")
print(qdrant_client.get_collections())
```

## Expected Behavior
QDrant client shouldn't throw exceptions

**Comment by generall** [comment]
fyi @joein

**Comment by IvanPleshkov** [comment]
Discord channel with discussion
https://discord.com/channels/907569970500743200/1133022847170777128

**Comment by joein** [comment]
Will look into it today

**Comment by joein** [comment]
Has already been fixed in #216 just need to make a package release

**Comment by IvanPleshkov** [comment]
Thanks for checking! Close as duplicate of fixed issue
